dev/io.openliberty.microprofile.telemetry.internal.common/resources/io/openliberty/microprofile/telemetry/internal/common/resources/MPTelemetry.nlsprops

```diff
@@ -55,6 +55,10 @@ CWMOT5005.mptelemetry.unknown.log.source=CWMOT5005W: The MicroProfile Telemetry
 CWMOT5005.mptelemetry.unknown.log.source.explanation=The Liberty runtime environment does not provide the specified log source.
 CWMOT5005.mptelemetry.unknown.log.source.useraction=Check the configuration information and specify one of the supported sources for MicroProfile Telemetry.
 
+CWMOT5006.tel.enabled.conflict=CWMOT5006W: Conflicting configuration for otel.sdk.disabled detected.
```

Comment:
I'm not convinced this error message has enough information. If possible we should at least tell the user where the conflicting configurations were found.